net-dialup/freeradius-client/files/freeradius-client-1.1.6-BJA-cross.diff

```diff
@@ -0,0 +1,67 @@
+--- configure.in.ori	2011-06-28 18:01:12.000000000 +0200
```

Comment:
patch should have a proper description (pretty much the same info git commits have)